src/phoenix/server/telemetry.py

```diff
@@ -1,41 +1,49 @@
 import os
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Literal, Optional
 
 from phoenix.config import (
+    ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_GRPC_ENDPOINT,
+    ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_HTTP_ENDPOINT,
     ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT,
     ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_HTTP_ENDPOINT,
 )
 
 if TYPE_CHECKING:
+    from opentelemetry.metrics import MeterProvider
     from opentelemetry.trace import TracerProvider
+
 from logging import getLogger
 
 logger = getLogger(__name__)
 
 
-def normalize_http_collector_endpoint(endpoint: str) -> str:
-    normalized_endpoint = endpoint
-    if not normalized_endpoint.startswith("http://") and not normalized_endpoint.startswith(
-        "https://"
-    ):
+def normalize_http_collector_endpoint(
+    endpoint: str,
+    suffix: Optional[Literal["v1/traces", "v1/metrics"]] = None,
+) -> str:
+    endpoint = endpoint.rstrip("/")  # remove trailing slashes
+    if not (endpoint.startswith("http://") or endpoint.startswith("https://")):
         logger.warning(
-            "HTTP collector endpoint should include the protocol (http:// or https://)."
-            "Assuming http."
+            "HTTP collector endpoint should include the "
+            "protocol http:// or https://. Assuming http."
         )
         # assume http if no protocol is provided
-        normalized_endpoint = f"http://{endpoint}"
-    if normalized_endpoint.endswith("/v1/traces"):
-        logger.warning(
-            "HTTP collector endpoint should not include the /v1/traces path. Removing it."
-        )
-        # remove the /v1/traces path
-        normalized_endpoint = normalized_endpoint[: -len("/v1/traces")]
-    # remove trailing slashes
-    normalized_endpoint = normalized_endpoint.rstrip("/")
-    return normalized_endpoint
+        endpoint = f"http://{endpoint}"
+    for tail in ("/v1/traces", "/v1/metrics"):
+        if endpoint.endswith(tail):
```

Comment:
Do we need to account for trailing slashes?

Reply:
This is removed on line 24.

Reply:
Missed that, thanks.